mava/networks.py

```diff
@@ -108,7 +110,54 @@ def __call__(self, obs_embedding: chex.Array, observation: Observation) -> distr
             jnp.finfo(jnp.float32).min,
         )
 
-        return distrax.Categorical(logits=masked_logits)
+        return tfd.TransformedDistribution(
+            distribution=tfd.Categorical(logits=masked_logits), bijector=tfb.Identity()
+        )
+
+
+class ContinuousActionHead(nn.Module):
+    """Continuous Action Head"""
+
+    action_dim: int
+    minimum: float = -1
+    maximum: float = 1
+
+    def setup(self) -> None:
+        self.actor_mean = nn.Dense(self.action_dim, kernel_init=orthogonal(0.01))
+        self.actor_log_std = self.param("log_std", nn.initializers.zeros, (self.action_dim,))
+        scale = 0.5 * (self.maximum - self.minimum)
+        self.bijector = tfb.Chain(
+            [tfb.Shift(self.minimum), tfb.Scale(scale=scale), tfb.Shift(1), tfb.Tanh()]
+        )
+
+    def _sigma_activation(
+        self, sigma: chex.Array, sigma_min: float = -4, sigma_max: float = 0
```

Comment:
In SAC I use `[-5, 2]`, but I know rlax uses these values, I don't really know which is better without running experiments, but I suspect it makes little difference.

But also can we rather make these values you can pass into the network and default them at the top :smile:

Reply:
Yeah maybe add them to hyperparams sweep and see what we can get 🤔 

Here I'm afraid we'll have different attributes for both discrete and continuous action heads, isn't the best way to add them all into a config (cfg.network) and access everything related to networks there (even the action_dim and min, max)??

Reply:
Ye so for sigma min/max it will just work with how hydra creates the networks, but I we definitely shouldn't pass in action_dim, act_min/max as these are runtime variables and we don't want to set them in the config. I think if we pass in env to all networks (discrete and continuous) then we should be able to get all the info we need